### ISSUE TITLE
Improve doctor text visibility by using darker color

### ISSUE THREAD

**Issue by ospirentos** [ospirentos/clinicamber#14]
Doctor-related texts on the site are currently hard to read due to insufficient contrast. Update the color of these texts to a darker shade to improve readability and accessibility for users.
